fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/Main.java

```diff
@@ -94,9 +94,6 @@
     // The thread pool used for concurrent request processing
     private ExecutorService pool;
 
-    // number of ms to wait for the thread pool to shut down
-    private long poolShutdownWaitTime = 60000;
-
     private List<FileExpectation> fileExpectations = new ArrayList<>();
 
     // mode of operation
```

Comment:
does this mean poolShutdownWaitTime has never been used?

Reply:
correct, eclipse warned that this value is never used and so i removed it.

however, to be completely honest, i was just looking for an edit i could make in jdbc to verify the conditional logic.